src/components/todoitem.js

```diff
@@ -0,0 +1,12 @@
+const TodoItem = ({id, done, title, onToggleTodo, onDeleteTodo}) => {
+	return (
+		<div>
+			<div key={id} className="TodoItem">
+				<input checked={done} onChange={() => onToggleTodo(id)} type="checkbox" />
+				<p>{title}</p>
+				<button onClick={() => onDeleteTodo(id)}>Löschen</button>
+			</div>
+		</div>
```

Comment:
Please think about which semantic elements could be suitable for your component. Maybe you need a `form` element? 
As your `TodoItem` can be part of a list you work with `ul` and `li`. Therefore, you would add a `ul` element in your `TodoList` component and the `div` element in row three of your `TodoItem` component would be a `li` element. A solution I would prefer. ;)
You also don't need to add a key attribute. To add the key prop your `TodoList` component is enough for React to work correctly.
And please check how you write accessible forms. :)
And you should add a type your delete button, as the default type is `submit`, but a delete button is not a submit button. If you enter `type="button"` it should be fine. 👍